src/user-agent/index.ts

```diff
@@ -0,0 +1,402 @@
+export const userAgentData = {
```

Comment:
can we explore using some external library for browser parsing

Reply:
Sure will look into it.

Reply:
@silentsakky, @madhavdhuppe had this idea, but we thought as we already have a proven solution, we should use it, as it would give us freedom to change it if necessary, but surely we can look at the alternatives and take a decision.